index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">      
+          <title> Tribute Page</title>
+          <link href="stylesheet"  rel="stylle.css">
```

Comment:
Kindly check the values of href and rel attributes of your css link and ensure that they contain the right values.
Also, kindly ensure that  the  name of your CSS file corresponds with the name in the href. This is necessary in order for your page to display its styles correctly.